.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 node_modules
 .DS_Store
+package-lock.json
```

Comment:
package-lock should remain committed to ensure every dev has the same exact dependencies for everytinh

Reply:
Oke, thanks!